components/builder-worker/src/runner/mod.rs

```diff
@@ -34,7 +34,10 @@ use bldr_core::logger::Logger;
 use chrono::Utc;
 use hab_core::os::users;
 use hab_core::package::archive::PackageArchive;
-use hab_core::util::perm;
+#[cfg(not(windows))]
```

Comment:
I think this should be `#[cfg(unix)]` rather than `not(windows)` since the `posix_perm` module depends on unix libraries, not just the absence of windows.

Reply:
I don't disagree, but the code that implements it uses `#[cfg(not(windows))]`. Shouldn't those match?

Reply:
I would argue that code should also be `#[cfg(unix)]`

Reply:
Yeah, that's a good argument :) There are several places that use it though. Would it be reasonable to do another PR changing all of those calls?